### ISSUE TITLE
Timeseries panel: Better defaults for range 0-1

### ISSUE THREAD

**Issue by freeseacher** [grafana/grafana#36858]
Hello

**What would you like to be added**:
most of SLI queries leads to numbers in range of 0-1. But now (8.0.6) drawing graph in that range leads to 
![image](https://user-images.githubusercontent.com/289277/125967868-35f0b6e9-26b3-470a-aaee-0bd3bbd37d1f.png)
looks weird but ok. probably we should switch units to percent to see the actual numbers. 
but 
![image](https://user-images.githubusercontent.com/289277/125968035-7be4383a-e552-4f7c-a792-50e9a3cc3159.png)
 :(
and only after switching decimal 
![image](https://user-images.githubusercontent.com/289277/125968159-845c3aff-e678-490d-8d04-eb962a126bcf.png)

**Why is this needed**:
Current behavior is weird and unexpected

**Comment by leeoniya** [comment]
i'm going to close this as a duplicate of https://github.com/grafana/grafana/issues/36448

the resolution to this issue (and related others) is rather more involved and tracked in https://github.com/grafana/grafana/issues/35838
